src/components/Book/Books.js

```diff
@@ -1,25 +1,34 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
-import { generate } from 'randomized-string';
-import { alphanumeric } from 'randomized-string/lib/types';
-import Book from './Book';
+import { useSelector, useDispatch } from 'react-redux';
+
+// import PropTypes from 'prop-types';
 import AddBooks from '../Addbooks/AddBooks';
-import './Books.css';
+import { removeBook } from '../../Redux/books/books';
 
 const Books = () => {
-  const book = useSelector((state) => state.booksReducer);
+  const books = useSelector((state) => state.booksReducer);
+  console.log(books);
+  const dispatch = useDispatch();
+
+  const handleClick = (e) => {
+    dispatch(removeBook(e));
+  };
 
   return (
     <>
       <main className="main">
         <section className="books-list-section">
+          <ul>
+            {books.map((book) => (
+              <li id={book.item_id} key={book.item_id}>
+                {book.title}
+                by
+                {book.author}
```

Comment:
- [ ] You should follow the design layout. Please change the design.

YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/168427010-999df03e-eda1-4c8c-ba41-22ccad582953.png)

ORIGINAL

![image](https://user-images.githubusercontent.com/38635158/168426986-c3e6fdb2-4847-483d-a69e-245eada85d97.png)